pq-crypto/bike/defs.h

```diff
@@ -43,6 +43,8 @@
 #define ELL_K_BITS  256ULL
 #define ELL_K_SIZE (ELL_K_BITS/8)
 
+#define BIKE1
```

Comment:
So I'm not attached either way, just a little curious why you switched to a define here rather than the makefile?